src/main/java/com/github/manolo8/darkbot/utils/LibUtils.java

```diff
@@ -0,0 +1,21 @@
+package com.github.manolo8.darkbot.utils;
+
+import java.nio.file.Paths;
+
+public class LibUtils {
+    /**
+     * @param library Filename of library without extension, e.g. "DarkBoatAPI"
+     * @return Absolute path of library with extension based on OS
+     */
+    public static String getLibPath(String library) {
+        return Paths.get("lib", library).toAbsolutePath().toString() + getExtension();
```

Comment:
```suggestion
        return Paths.get("lib", library + getExtension()).toAbsolutePath().toString();
```
it's the file that has an extension, not an appended thing at the end of the path